src/accounts.rs

```diff
@@ -42,6 +50,17 @@ pub(crate) fn cancel_positions(client: &Client) -> Result<(), Error> {
     Ok(())
 }
 
+/// Determine whether an account exists under an account family and find the account family code.
+pub(crate) fn family_codes(client: &Client) -> Result<impl Iterator<Item = FamilyCode> + '_, Error> {
+    client.check_server_version(server_versions::ACCOUNT_SUMMARY, "It does not support fammily code requests.")?;
```

Comment:
typo family

Reply:
Server version for this should be `server_version::REQ_FAMILY_CODES`

Reply:
Will change accordingly.  Thanks